docs/content/direct/core-chart.md

```diff
@@ -163,6 +186,36 @@ helm upgrade --install ks-core oci://ghcr.io/kubestellar/kubestellar/core-chart
   --set-json='WDSes=[{"name":"wds1"}]'
 ```
 
+The core chart also supports the use of a pre-existing cluster (or any space, really) as an ITS. A specific application is to connect to existing OCM clusters. As an example, create a first local kind cluster with OCM installed in it:
+
+```shell
+kind create cluster --name ext1
+
+clusteradm init
+```
+
+Then, create a second kind cluster suitable for KubeStellar installation and create a bootstrap secret in the new cluster with the kubeconfig information of the `ext1` cluster:
+
+```shell
+bash <(curl -s https://raw.githubusercontent.com/kubestellar/kubestellar/v$KUBESTELLAR_VERSION/scripts/create-kind-cluster-with-SSL-passthrough.sh) --name kubeflex --port 9443
+
+bash <(curl -s https://raw.githubusercontent.com/kubestellar/kubestellar/v$KUBESTELLAR_VERSION/scripts/create-external-bootstrap-secret.sh) --controlplane its1 --context kind-ext1 --address https://ext1-control-plane:6443 --verbose
+```
+
+Note that the last command above creates a secret named `its1-bootstrap` in the `default` namespace of the `kind-kubeflex` cluster.
+
+The `--address` URL needs to be one that the KubeFlex controller can use to open a connection to the external cluster's Kubernetes apiserver(s). In this example, the external cluster is a kind cluster with one kube-apiserver and it listens on port 6443. This example relies on the DNS resolver in Docker networking to map the domain name `ext1-control-plane` to the Docker network address of the container of that same name.

```

Comment:
Oops, I forgot one critical detail. That kube-apiserver listens on port 6443 _in its node's network namespace_.